rir/src/compiler/pir_tests.cpp

```diff
@@ -394,7 +394,106 @@ bool testPir2Rir(const std::string& name, const std::string& fun,
     return checkPir2Rir(orig, after);
 }
 
+class MockBB : public BB {
+    class MockCode : public pir::Code {
+      public:
+        MockCode(BB* e, size_t s) : Code() {
+            entry = e;
+            nextBBId = s;
+        }
+        ~MockCode() { entry = new MockBB; }
```

Comment:
???

Reply:
:) don't ask. `~Code` insists on deleting something

Reply:
Now that I looked at the [code](https://github.com/reactorlabs/rir/blob/master/rir/src/compiler/pir/code.cpp#L28) it makes sense. Please add a comment, because by itself, this line is very 😖 .